code/modules/clothing/shoes/boots.dm

```diff
@@ -205,3 +205,9 @@
 	icon_state = "kala_boots"
 	item_state = "kala_boots"
 	contained_sprite = TRUE
+/obj/item/clothing/shoes/jackboots/falerio
```

Comment:
are these like specific to falerio in any way? the desc or name does not really read as if they were
and like they're just boots

imo these should have a path like `/obj/item/clothing/shoes/jackboots/gaiters`, and exist with other generic jackboots

Reply:
I'll see what Schwann wants. I had them in boots to begin with I think, but that didn't make it clear regarding what they actually are - they're supposed to have lore significance but be open to whomever